lib/auth/auth_with_roles.go

```diff
@@ -3479,8 +3479,7 @@ func (a *ServerWithRoles) trySettingConnectorNameToPasswordless(ctx context.Cont
 		return trace.Wrap(err)
 	}
 
-	// Only set the connector name on the first user registration.
-	if len(users) != 1 {
+	if !hasOneNonPresetUser(users) {
```

Comment:
Add a comment explaining why having exactly one non-preset user important, or maybe restore the previous comment?

Reply:
Yes, that's a good idea, ty!